app/controllers/create.rb

```diff
@@ -0,0 +1,50 @@
+get '/create/routine' do
+  @exercises = Exercise.all
+  erb :create
+end
+
+post '/create/routine' do
+  @user = User.find(session[:user_id])
+  @routine = Routine.create(name: params[:name], sets: params[:sets], reps: params[:reps], weight: params[:weight], user_id: session[:user_id])
```

Comment:
You need to check to see if this create worked. Either use `create!` or check the return value of `create`